packages/config/src/ConfigSchema.ts

```diff
@@ -175,17 +199,140 @@ export class ConfigSchema {
 
     /**
      * Loads Imperative profile schema objects from a schema JSON file.
-     * @param schemaJson The schema JSON for config
+     * @param schema The schema JSON for config
      */
-    public static loadProfileSchemas(schemaJson: IConfigSchema): IProfileTypeConfiguration[] {
-        const patternName = Object.keys(schemaJson.properties.profiles.patternProperties)[0];
+    public static loadSchema(schema: IConfigSchema): IProfileTypeConfiguration[] {
```

Comment:
This is the `next-only` breaking change mentioned in the PR comments.
Not really needed, but I thought it was nice to be consistent with the API names 😋